docs/molgenis/semantics.md

```diff
@@ -5,11 +5,16 @@ For this you use the `semantics` field in EMX2.
 
 ## Formatting
 
-There are 2 formats allowed for this field:
-* an IRI surrounded by `<` and `>` (f.e. `<http://purl.org/dc/elements/1.1/title>`)
-* a prefixed name using a defined prefix followed by the relative IRI (f.e. `dcterms:title` if `dcterms` has been defined)
+There are 2 types of data allowed in this field:
+* an IRI (f.e. `http://purl.org/dc/terms/title`)
+* a prefixed name (f.e. `dcterms:title`), assuming that prefix [is defined](#defining-namespaces)
```

Comment:
great, thanks

Reply:
np 🙂 

Will wait with merge till after #4700 is merged so that docs stay in sync with code.